src/supermarket/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2'
 services:
   db:
-    image: postgres:9.3
+    image: postgres:13.4
```

Comment:
We should use the 13 tag so we don't have to update this every month

Reply:
@tas50 shouldn't it be in sync with the exact version we are using?

Reply:
We're always going to update to the latest since every version has a CVE fix in it

Reply:
ok, will keep it as 13

Reply:
updated

Reply:
I think we should specify the minor version also in case if there is a bug in the latest minor version we have not tested.

Reply:
Reverting to specifying minor version. Will keep all the versions in sync for future releases. @tas50 @saghoshprogress

Reply:
specified with minor version.